src/pages/SplashPage/SplashPage.tsx

```diff
@@ -0,0 +1,46 @@
+import "./SplashPage.scss";
+import Button from "../../components/Button/Button";
+import paintingWoman from "../../images/paintingWoman.png";
```

Comment:
The image file name should be lowercase hyphenated `paintingWoman.png` -> `painting-woman.png`